assests/styleSheets/general.css

```diff
@@ -0,0 +1,13 @@
+@import "./reset.css";
+@import "./variables.css";
+
```

Comment:
Please move the styles to the main CSS file.

Reply:
my "main" file is for setting the basic designs like main contents and main boxes but general is for some general tags like body heading and ... . 
I thought it would be more clear if separate them.

Reply:
ok perfect. resolving.